pages/_document.tsx

```diff
@@ -1,79 +1,51 @@
-/* eslint-disable react/jsx-props-no-spreading */
-import Document, {
-  DocumentContext, Head, Html, Main, NextScript,
+import {
+  Html, Head, Main, NextScript,
 } from 'next/document'
-import { ServerStyleSheet } from 'styled-components'
 
-export default class MyDocument extends Document {
-  render() {
-    return (
-      <Html>
-        <Head>
-          <link
-            href="/fonts/Anonymous_Pro/AnonymousPro-Regular.tff"
-            rel="preload"
-            as="font"
-            type="font/tff"
-            crossOrigin="anonymous"
-          />
-          <link
-            href="/fonts/Anonymous_Pro/AnonymousPro-Italic.tff"
-            rel="preload"
-            as="font"
-            type="font/tff"
-            crossOrigin="anonymous"
-          />
-          <link
-            href="/fonts/Anonymous_Pro/AnonymousPro-Bold.tff"
-            rel="preload"
-            as="font"
-            type="font/tff"
-            crossOrigin="anonymous"
-          />
-          <link
-            href="/fonts/Source_Serif_4/SourceSerif4-VariableFont_opsz,wght.tff"
-            rel="preload"
-            as="font"
-            type="font/tff"
-            crossOrigin="anonymous"
-          />
-          <link
-            href="/fonts/Source_Serif_4/SourceSerif4-Italic-VariableFont_opsz,wght.tff"
-            rel="preload"
-            as="font"
-            type="font/tff"
-            crossOrigin="anonymous"
-          />
-        </Head>
-        <body>
-          <Main />
-          <NextScript />
-        </body>
-      </Html>
-    )
-  }
-
-  static async getInitialProps(ctx: DocumentContext) {
-    const sheet = new ServerStyleSheet()
-    const originalRenderPage = ctx.renderPage
-
-    try {
-      ctx.renderPage = () => originalRenderPage({
-        enhanceApp: (App) => (props) => sheet.collectStyles(<App {...props} />),
-      })
-
-      const initialProps = await Document.getInitialProps(ctx)
-      return {
-        ...initialProps,
-        styles: (
-          <>
-            {initialProps.styles}
-            {sheet.getStyleElement()}
-          </>
```

Comment:
The merge went wrong here. We need to generate the CSS code here.

Reply:
Meaning restore row 56-78?